.github/workflows/publish-chrome-development.yml

```diff
@@ -0,0 +1,106 @@
+name: publish-chrome-development
+on:
+  release:
+    types: [prereleased]
+  workflow_dispatch:
+    inputs:
+      attemptNumber:
+        description: "Attempt number"
+        required: false
+        default: "1"
+      maxAttempts:
+        description: "Max attempts"
+        required: false
+        default: "10"
```

Comment:
10 max attempts seem excessive tbh

Reply:
The reason it's this long is because it's a matter of waiting for the the previous version to be approved on the Web Store, which can take days. In particular it's 12 hrs delay * 10 times, so 5 days total, which I think is reasonable for this.

Reply:
Ah it has to wait for approval, I didn't know this. Thanks!